src/meta/windows_attributes.rs

```diff
@@ -0,0 +1,126 @@
+#[cfg(windows)]
```

Comment:
maybe we can drop all the `#[cfg(windows)]` in this file, if we set it in `mod.rs`